contrib/install/install.sh

```diff
@@ -0,0 +1,210 @@
+#!/bin/bash
+#
+#
+#  Copyright 2016 CUBRID Corporation
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+function get_cubrid_dir() {
+  cubrid_str=/CUBRID/
+  cubrid_str_len=${#cubrid_str}
+  current_dir="`pwd`"
+
+  find_dir=$(echo ${current_dir} | grep ${cubrid_str})
+
+  if [ -z "$find_dir" ];then
+    echo ""
+  else
+    temp_dir=${current_dir%%$cubrid_str*}
+    temp_dir_len=${#temp_dir}
+    cubrid_dir_len=`expr $temp_dir_len + $cubrid_str_len`
+    cubrid_dir=$(echo $current_dir | cut -c1-$cubrid_dir_len)
+    echo "$cubrid_dir"
+  fi
+}
+
+
+cubrid_home=$(get_cubrid_dir)
+
+if [ -z "$cubrid_home" ];then
+  cubrid_home="`pwd`"
+fi
+
+echo "Is CUBRID installed in the directory below? [Yn]:"
+echo "\"$cubrid_home\""
+
+read line leftover
+is_installed_dir=TRUE
+
+case ${line} in
+  n* | N*)
+    is_installed_dir=FALSE
+esac
+
+if [ "x${is_installed_dir}x" = "xFALSEx" ];then
+  echo ""
+  echo "Please enter the directory where CUBRID is installed: "
+
+  read input_dir leftover
+  cubrid_home=${input_dir}
+fi
+
```

Comment:
How about checking for existence of CUBRID home before proceed ?

```
if [ ! -d $cubrid_home ];then
  echo "$cubrid_home: no such directory"
  exit
fi
```

Reply:
Thanks for your advice.